### ISSUE TITLE
io.grpc.StatusRuntimeException: UNKNOWN: EOF 

### ISSUE THREAD

**Issue by gangxie112** [grpc/grpc-java#9265]
I'm using the grpc-java as grpc client to call golang grpc server. It's bidi stream。 After the client called oncomplete, and the server closed the stream directly. on client side, unknown eof exception is received. is it FAD? I think the grpc-java should handle this EOF after the oncomplete is called？

**Comment by sergiitk** [comment]
Yes, grpc-java should be interoperable with grpc-go. Could you please provide more information so we can reproduce the issue:

- What grpc-java and grpc-go versions are you using?
- Would it be possible for you to create a small repo  to demonstrate this issue? You could use routeguide as a startpoint, it's available in [java](https://github.com/grpc/grpc-java/tree/master/examples/src/test/java/io/grpc/examples/routeguide) and [go](https://github.com/grpc/grpc-go/tree/master/examples/route_guide).
- What dod you mean by FAD?

**Comment by ejona86** [comment]
I strongly suspect this can be answered by grpc-go. The way error codes work I think the server may be misusing the grpc API and an EOF error is trying to be sent. In particular, grpc-java wouldn't construct an UNKNOWN error with description "EOF."

Transferring this to the grpc-go repo.
